cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/inmemory/InMemoryStreamAdmin.java

```diff
@@ -41,8 +41,8 @@ public InMemoryStreamAdmin(InMemoryQueueService queueService) {
   }
 
   @Override
-  public void dropAll() throws Exception {
```

Comment:
I'm not sure its confirmed, but this is what made sense to me. (It wouldn't need to be used anywhere except something like unrecoverable/reset).
Maybe its worth brining up whether something like that should exist for CDAP now that there are multiple namespaces.